src/use_case/send_notification/NotificationOutputData.java

```diff
@@ -0,0 +1,11 @@
+package use_case.send_notification;
+public class NotificationOutputData {
+    private final String gptResponse;
+    public NotificationOutputData(String gptResponse) {
```

Comment:
I'm assuming gptResponse would be the extracted String from the interactor, maybe a comment describing if the string is extracted and ready to be directly inputted into gpt would be nice